src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -162,6 +162,7 @@ var ReactDOMSelect = {
     return Object.assign({}, DisabledInputUtils.getNativeProps(inst, props), {
       onChange: inst._wrapperState.onChange,
       value: undefined,
+      defaultValue: undefined,
```

Comment:
This copies the behavior in `ReactDOMInput.js`, and is needed to pass unit tests that attempt to render selects without a markup mismatch.